### PR TITLE
ZEPPELIN-3029. Cannot delete an interpreter

### PR DESCRIPTION
**Pull request by zjffdu**
### What is this PR for?

Only copy interpreterSetting from interpreterSettingTemplates when zeppelin is started the first time (interpreter.json doesn't exist)

### What type of PR is it?
[Bug Fix]

### Todos
* [ ] - Task

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-3029

### How should this be tested?
* Manually verified.  Delete an interpreter, and then restart zeppelin to check.

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by zjffdu** [comment]
@prabhjyotsingh Could you help review it ? Thanks

File: zeppelin-zengine/src/main/java/org/apache/zeppelin/interpreter/InterpreterSettingManager.java [hunk 1]
```diff
@@ -172,6 +172,19 @@ private void loadFromFile() {
...
+        interpreterSetting.setInterpreterSettingManager(this);
+        interpreterSetting.setAngularObjectRegistryListener(angularObjectRegistryListener);
+        interpreterSetting.setRemoteInterpreterProcessListener(
+            remoteInterpreterProcessListener);
+        interpreterSetting.setAppEventListener(appEventListener);
+        interpreterSetting.setDependencyResolver(dependencyResolver);
+        interpreterSetting.setLifecycleManager(lifecycleManager);
+        interpreterSetting.postProcessing();
+        interpreterSettings.put(interpreterSetting.getId(), interpreterSetting);
+      }
```
Comment (felixcheung) [inline zeppelin-zengine/src/main/java/org/apache/zeppelin/interpreter/InterpreterSettingManager.java]: how about these?
```       .setGroup(group)
         .setName(group)
         .setInterpreterInfos(interpreterInfos)
         .setProperties(properties)
         .setDependencies(new ArrayList<Dependency>())
         .setOption(option)
         .setInterpreterDir(interpreterDir)
         .setRunner(runner)
```

**Comment by zjffdu** [comment]
Will merge it if no more discussion

**Comment by prabhjyotsingh** [comment]
Tested locally, works well. LGTM.

**Pull request closed without merging**